examples/jenkins/pipeline/README.md

```diff
@@ -0,0 +1,70 @@
+This set of files will allow you to deploy a Jenkins server that is capable of executing Jenkins pipelines and
+utilize pods run on OpenShift as Jenkins slaves.
+
+To walk through the example:
+1) Stand up an openshift cluster from origin master, installing the standard imagestreams to the openshift namespace:
```

Comment:
@bparees you need empty line before `1)` otherwise markup will join it with previous line